Vladislav Pozniak/0/run.rb

```diff
@@ -0,0 +1,79 @@
+require 'roo'
+require 'roo-xls'
+
+# Level 1
+current = Dir.glob('./data/*').max
+table = Roo::Spreadsheet.open(current, extension: File.extname(current))
+
+print "What price are you looking for?\n>\s"
+item = gets.chomp
+
+found = 0
```

Comment:
useless assignment

use Hash like
```Ruby
{
 product_name => price
}
```

that will make you data safe and pretty good. and you can replace `found` to `products.size`